src/core/execute-serialize.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#include "execute.h"
+
+/* Note that the Unit object is only minimally serialized, to allow for logging with log_unit* in the
+ * deserializer. Only type, id, cgroup_id and invocation_id are serialized and deserialized. */
+
+int exec_serialize(FILE *f,
```

Comment:
hmm, so hmm, i wonder if we should give this a more precise name, i.e. exec_serialize_invocation() or so, as this serializes a complete invocation, i.e. the combination of the various objects that encapsulate one invocation request.

we have multiple purposes of serialiation now in the mix after all: the regular serialization of *state* that we do on daemon reexec/reload. And this serialization of an invocation request, and we really should underline this difference somehow, to keep them apart.

maybe even rename the exsiting unit_serialize() call to unit_serialize_state() to make this clearer.

Also, please add comments to both functions underlining the difference. Both after all operate on behalf of a unit, but they focus on different parts to serialize.